Resources/Prototypes/Entities/Objects/Devices/Syndicate_Gadgets/camera_bug.yml

```diff
@@ -0,0 +1,25 @@
+- type: entity
+  id: CameraBug
+  parent: [ BaseItem, BaseSyndicateContraband ]
+  name: Camera bug
+  description: personal camera viewing device
```

Comment:
Can you give it a proper item description for the examine window?
For example `An illegal syndicate device that allows you to hack into the station's camera network.`